tests/ops/functions/test_eigvals.py

```diff
@@ -97,9 +97,10 @@ def test_parametric_gates_qnode(self, op_class):
     @pytest.mark.parametrize("op_class", one_qubit_one_parameter)
     def test_adjoint(self, op_class):
         """Test that the adjoint is correctly taken into account"""
+        rounding_precision = 6
         res = qml.eigvals(qml.adjoint(op_class))(0.54, wires=0)
         expected = op_class(-0.54, wires=0).eigvals()
-        assert np.allclose(res, expected)
+        assert set(np.around(res, rounding_precision)) == set(np.around(res, rounding_precision))

```

Comment:
What happened here?

Reply:
The order is flipped.  
```
>>> res
[(0.9637708963658903-0.266731436688831j),
 (0.9637708963658904+0.26673143668883115j)]
>>> expected
array([0.9637709+0.26673144j, 0.9637709-0.26673144j])
```

Reply:
Why? Was the test wrong before?

Reply:
`np.linalg.eigvals` does not return the eigenvalues in a particular order.